MIGRATING.md

```diff
@@ -0,0 +1,71 @@
+# Okta Java Management SDK Migration Guide
+ 
+This SDK uses semantic versioning and follows Okta's [library version policy](https://developer.okta.com/code/library-versions/). In short, we don't make breaking changes unless the major version changes!
+
+## Migrating from 1.x.x to 2.0.0
+
+Version 2.0.0 of this SDK introduces a number of breaking changes from previous versions. 
+In addition to many new classes/interfaces, some existing class/interfaces are no longer backward compatible due to method renaming and signature changes, see [Breaking Changes](#breaking-changes).
+
+### Breaking Changes
+
+Following is a list of changes that break backward compatibility in version 2.0.0.
+
+- `com.okta.sdk.client.Client`
+   - Renamed `createRule` to `createGroupRule`
+   - Renamed `getRule` to `getGroupRule`
+   - Renamed `listRules` to `listGroupRules`
+   
+- `com.okta.sdk.resource.log.LogEventList` 
+   - `getLogs(String until, String since, String filter, String q, String sortOrder)`
+   Signature changed to `getLogs(Date since, Date until, String filter, String q, String sortOrder)`
+   
+- `com.okta.sdk.resource.group.GroupList` 
+   - `listGroups(String q, String filter, String expand)`
+   Signature changed to `listGroups(String q, String filter)`
```

Comment:
is the object automatically expanded? did this change on the Okta side?

Reply:
Not quite sure on this one. My guess is the same.

Reply:
@bretterer if you have any background on why `expand` was removed, could you please share here?

Reply:
`expand` is not documented, so we don't want to have this in the SDK at this time.  Suggest the developer who needs this makes the call to the api themselves with this, but warn that `expand` option could go away at any time

Reply:
Two options come to mind:
1. Instruct the developer to make a second API call to the `group API` (i'm not sure this actually returns the same result though), but this would be ideal, as that `expand` option is undocumented and could be removed at any time.
1. provide an example using this technique: https://github.com/okta/okta-sdk-java#call-other-api-endpoints.

Reply:
updated